### ISSUE TITLE
Loading video of 3min and process it with batch manager (only 8Gb of VRAM) 

### ISSUE THREAD

**Issue by rafterradillos** [Kosinkadink/ComfyUI-VideoHelperSuite#182]
Hello guys,

I wanted to load a video of 3min and process 20 frames by 20 frames using VHS suite with batch manager set on 20 frames (I have only 8Gb Vram). But each 20 frames, the next image is not completely linked to the others as if the seed changed (but i set it on "fixed") so you can see it's not completely smooth. Do you have any solutions to process a long video ?

Thank you for your help !

This is my workflow [workflow (5).json](https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/files/14780153/workflow.5.json)

And this is the video output
https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/assets/58289413/05c1c8a2-baa6-419c-9e9b-b6d09745ac8a

**Comment by Kosinkadink** [comment]
If you're trying to use Batch Manager with AnimateDiff, you can't. You have to load all frames. VRAM usage should not be affected - only context_length amount of frames will be processed at once. You can use Batched VaeEncode/Batches VAEDecode nodes that are present in VHS if VaeEncode/VaeDecode doesn't work as intended.

The Batch Manager is basically a meta-batch manager, where it basically queues multiple separate runs as if you were to press Queue Prompt multiple times. @AustinMroz would you be fine if we rename Batch Manager to Meta-Batch Manager? Just to make it obvious it's not the same thing as normal batches.

**Comment by AustinMroz** [comment]
I agree that a name change is warranted. As is an update to the readme.

I can also corroborate that this is the big downside of the Batch Manager. I'd like to see it improved upon (a system for sending a number of overlap frames?), but don't have high hopes for anything being feasible.
